src/main/java/seedu/address/ui/FileExplorerPanel.java

```diff
@@ -53,6 +55,20 @@ public void updateCurrentPath() {
         currentPathDisplay.setText(currentPath.getAddress().value);
     }
 
+    /**
+     * Changes the current directory to the parent directory.
+     */
+    public void cdToParentPath() {
+        MainWindow mainWindow = MainWindow.getInstance();
+
+        String command = "cd ../";
+        try {
+            mainWindow.executeCommand(command);
+        } catch (CommandException | ParseException exception) {
+            // do nothing
```

Comment:
Should you do nothing here? Maybe indicate the error by changing the text in ResultDisplay?

Reply:
I think the error here is just for UI to react to, like for AB3, when a command is wrong, it makes the text red. But currently I think there is no need to do that, so I think it's fine to do nothing here

Reply:
<img width="722" alt="屏幕截图 2020-10-27 223153" src="https://user-images.githubusercontent.com/68622434/97315991-3f432400-18a4-11eb-993b-80d496aacc8d.png">
This is the try and catch for the old AB3 command box, I commented it out, since we are not changing the color there, so, currently it's also doing nothing